ui/app/src/views/projects/ProjectTabs.tsx

```diff
@@ -99,7 +103,16 @@ function TabButton(props: TabButtonProps) {
     case dashboardsTabIndex:
       return (
         <>
-          <CRUDButton text="Add Dashboard" variant="contained" onClick={() => setCreateDashboardDialogOpened(true)} />
+          <Box>
+            <Stack direction="row" spacing={1}>
+              <CRUDButton
+                text="Add Dashboard"
+                variant="contained"
+                onClick={() => setCreateDashboardDialogOpened(true)}
+              />
+              <CRUDButton color="inherit" text="Explore" variant="contained" onClick={handleGoToExplore} />
```

Comment:
The color in light theme is strange, no?

Reply:
A bit!
Any suggestion?

Reply:
WDYT about secondary?
<img width="1840" alt="image" src="https://github.com/perses/perses/assets/8205546/acf25901-4984-4ab7-891f-bd1dae189ce6">